### ISSUE TITLE
Copy/paste as JSX

### ISSUE THREAD

**Issue by Wolfr** [usebedrock/bedrock#388]
What if we have a transform on the HTML example where it transforms the HTML to JSX, to then be able to c/p as JSX? This would then make it easier to copy/paste to React projects.

```
export default function Example() {
  return (
    <div className="something">

    </div>
  )
}
```

Ref https://magic.reactjs.net/htmltojsx.htm

**Comment by Wolfr** [comment]
A version of this was implemented in https://github.com/usebedrock/bedrock/tree/feature/jsx-copy .

**Comment by Wolfr** [comment]
This is now in Bedrock canary branch, along with a fix for #387 .
